pal/screen-adapter/minigame/screen-adapter.ts

```diff
@@ -80,11 +80,19 @@ class ScreenAdapter extends EventTarget {
             screenWidth = sysInfo.screenWidth;
             screenHeight = sysInfo.screenHeight;
         }
-        if (ALIPAY && rotateLandscape  && screenWidth < screenHeight) {
+        if (ALIPAY && rotateLandscape && screenWidth < screenHeight) {
             const temp = screenWidth;
             screenWidth = screenHeight;
             screenHeight = temp;
         }
+        //Fix: wrong screen size on TAOBAO_MINIGAME iPad device
+        if (TAOBAO_MINIGAME) {
```

Comment:
The comment says it is fixed on iPad, but this condition takes effect on all devices.